index.js

```diff
@@ -80,23 +82,46 @@ fetch(apiUrl)
         iconArray.push(data.list[2].weather[0].icon);
         iconArray.push(data.list[10].weather[0].icon);
         iconArray.push(data.list[18].weather[0].icon);
-    })
 
-/* tried to fetch api url from flicker for changing background images,but encountering with an error */
+        function createIcon() {
+            let iconSet = document.querySelectorAll("ul > li");
+            for (let i = 0; i < iconArray.length && i < iconSet.length; i++) {
+                let newIconEle = document.createElement("span");
+                newIconEle.classList.add("iconStyle");
+                let imageUrl = "http://openweathermap.org/img/wn/" + iconArray[i] + "@2x.png";
+                newIconEle.innerHTML = "<img src=" + imageUrl + ">";
+                iconSet[i].appendChild(newIconEle);
+            }
+        }
+        createIcon();
+    });
 searchBtnCmp.addEventListener("click", function() {
-
-    const cityUrl = "http://api.flickr.com/services/rest/?method=flickr.photos.search&api_key=528fc5e2ec09109cd0696b0555e7bc81&tags=" + weatherSearchBar.value + "&format=json&nojsoncallback=1";
+    const proxyurl = "https://cors-anywhere.herokuapp.com/";
+    const cityUrl = "http://api.flickr.com/services/rest/?method=flickr.photos.search&api_key=ef3d8a6207b554e853fcfe4b900bd9b7&tags=" + weatherSearchBar.value + "&format=json&nojsoncallback=1";
     console.log(cityUrl);
-    fetch(cityUrl)
+    fetch(proxyurl + cityUrl)
         .then(response => response.json())
         .then(data => {
 
-            let farmId = photos.photo[0].id;
-            let serverId = photos.photo[0].server;
-            let id = photos.photo[0].id;
-            let secretId = photos.photo[0].secret;
+            let farmId = data.photos.photo[0].id;
+            let serverId = data.photos.photo[0].server;
+            let id = data.photos.photo[0].id;
+            let secretId = data.photos.photo[0].secret;
             let backgroundImageUrl = "https://farm" + farmId + ".staticflickr.com/" + serverId + "/" + id + "_" + secretId + ".jpg";
             console.log(backgroundImageUrl);
             document.getElementsByClassName("leftContainer").css('background-image', 'url(' + backgroundImageUrl + ')');
         })
-});
\ No newline at end of file
+});
+let i = 0;
```

Comment:
you created this iterator in global scope. If you really want to do it then you need to create this variable inside `typeWriter` function. And those all are constants.

In general - this kind of motions are a bit disturbing and takes user focus from the application. As far as I know, it's better from UX perspective to make your application as static as possible, so everything looks predictive.
This is just a suggestion, you don't need to change it, only the variables :)